index.html

```diff
@@ -75,112 +89,167 @@ <h2>Multi-Post-Stories</h2>
+                              personalized reads; no accounts or <br> 
+                              sign-ups required. Has been the <br> 
+                              industry's standard.
+                          </p>
+                          <div class="btn">
+                              <ul>
+                                  <li>HTML</li>
+                                  <li>Bootstrap</li>
+                                  <li>Ruby</li>
+                              </ul>
+                          </div>
+                          <div class="see-btn">
+                              <button>See Projects</button>
+                          </div>
+                    </div>
+                    
+                  </div>
+                  <div class="project-section third desk-btn">
+                    <div class="project-section-box">
+                        <h2>Website Portfolio</h2>
+                        <p>
+                            A daily selection of privately  <br> 
+                            personalized reads; no accounts or <br> 
+                            sign-ups required. Has been the <br> 
+                            industry's standard.
+                        </p>
+                        <div class="btn">
+                            <ul>
+                                <li>HTML</li>
+                                <li>Bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <div class="see-btn">
+                            <button>See Projects</button>
+                        </div>
+                    </div>
+                      
+                  </div>
+              </div>
         </div>
     </section>
     <section class="about-me-section">
-        <div class="about-me">
-            <h1>About me</h3>
-            <p>
-                Hello I’m a software developer! I can help <br>
-                you build a product , feature or website <br>
-                Look through some of my work and <br> 
-                experience! If you like what you see and 
-                <br> have a project you need coded, don’t 
-                <br> hesitate to contact me.
-            </p>
-            <div class="resume-btn">
-                <button>Get my resume</button>
+            <div class="about-me">
+                <h1>About me</h3>
+                <p>
+                    Hello I’m a software developer! I can help <br>
+                    you build a product , feature or website <br>
+                    Look through some of my work and <br> 
+                    experience! If you like what you see and 
+                    <br> have a project you need coded, don’t 
+                    <br> hesitate to contact me.
+                </p>
+                <div class="resume-btn">
+                    <button>Get my resume</button>
+                </div>
```

Comment:
- [ ] The backgroud image on the figma is missing on your design. Kindly look into it and fix.

### Your Design
![your-about-me](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/f5874b10-a741-474d-be7d-602e01bff40f)

### Figma Design
![temp-about-me](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/a95ee04e-c715-4ff2-a5df-1dc845b71ef6)